ote/src/clj/ote/services/login.clj

```diff
@@ -27,16 +28,22 @@
 
 (defqueries "ote/services/login.sql")
 
-(defn- with-auth-tkt [response auth-tkt-value domain]
-  (update response :headers
-          assoc "Set-Cookie" (if (nil? domain)
-                               [(str "auth_tkt=" auth-tkt-value "; Path=/; HttpOnly")]
-                               ;; Three cookies are required to match ckan cookie configuration
-                               [(str "auth_tkt=" auth-tkt-value "; Path=/; HttpOnly; Secure")
-                                (str "auth_tkt=" auth-tkt-value "; Path=/; HttpOnly"
-                                     "; Domain=" domain "; Secure")
-                                (str "auth_tkt=" auth-tkt-value "; Path=/; HttpOnly"
-                                     "; Domain=." domain "; Secure")])))
+(defn- unix-epoch []
```

Comment:
Could be just `(def ^:private unix-epoch ...)` since ability to call this doesn't produce alternate values over time.

Reply:
Fixed :)